README.md

```diff
@@ -1,12 +1,42 @@
 # terraform-provider-infoblox
 Infoblox Provider for Terraform
 
-## Prerequisite 
+## Requirments
 
-* go
-* golang/dep
-* terraform
+* [Terraform](https://www.terraform.io/downloads.html) 
+* [Go](https://golang.org/doc/install) 1.9(to build the provider plugin)
+* [dep](https://github.com/golang/dep)
 
-## How to compile and build plugin
+## Building the Provider
 
-Just run *make build*
+clone repository to ```$GOPATH/src/github.com/infobloxopen/terraform-provider-infoblox```
+```
+$ mkdir -p $GOPATH/src/github.com/infobloxopen; cd $GOPATH/src/github.com/infobloxopen
+$ git clone git@github.com:infobloxopen/terraform-provider-infoblox
+```
+Enter the provider directory and build the provider
+```
+$ cd $GOPATH/src/github.com/infobloxopen/terraform-provider-infoblox
+$ make build
+```
+
+## Developing the Provider
```

Comment:
Building and Developing are interchanged here. 

Developing should have Building in it.